stage0/layout.ld

```diff
@@ -75,6 +75,13 @@ SECTIONS {
         bss_size = . - bss_start;
     } > ram_low
 
+    .data : 
+    AT ( ADDR (.text) + SIZEOF (.text) ) {
```

Comment:
Could you do `AT(text_end)` or something similar? Or I guess `text_end` might not have a value at this point yet...

All in all, this feels like it'll be quite fragile, but I don't know of a better way how to do it from the top of my head either.

Reply:
I could not find a better way that worked yet. I agree that it is a bit fragile. If I think of a better way I will fix it in a future PR.